configs/next/packages/gcc/stage_3

```diff
@@ -120,36 +147,86 @@ atcfg_configure() {
 			--without-libelf \
 			--with-as="${at_dest}/bin/as" \
 			--with-ld="${at_dest}/bin/ld" \
+			--with-stage1-ldflags="-Wl,--dynamic-linker=${ldso}" \
 			--with-cpu=${build_base_arch} \
 			--with-tune=${build_optimization}
 	else
-		echo "No configure on gcc_3 cross stage"
+		# Configure command for cross builds
+		CC="${system_cc}" \
+		CXX="${system_cxx}" \
+		LD_FOR_TARGET="${at_dest}/bin/${target}-ld" \
+		AS_FOR_TARGET="${at_dest}/bin/${target}-as" \
+		AR_FOR_TARGET="${at_dest}/bin/${target}-ar" \
+		NM_FOR_TARGET="${at_dest}/bin/${target}-nm" \
+		RANLIB_FOR_TARGET="${at_dest}/bin/${target}-ranlib" \
+		${ATSRC_PACKAGE_WORK}/configure \
+			--build=${host} \
+			--host=${host} \
+			--target=${target64:-${target}} \
+			--with-cpu=${default_cpu} \
+			--prefix=${at_dest} \
+			${with_longdouble:+--with-long-double-128} \
+			${secure_plt:+--enable-secureplt} \
+			${disable_multilib:+--disable-multilib} \
+			--with-advance-toolchain=$(basename ${at_dest}) \
+			--with-local-prefix=${at_dest} \
+			--enable-languages=c \
+			--enable-__cxa_atexit \
+			--enable-threads=posix \
+			--disable-libmudflap \
+			--disable-libssp \
+			--disable-libgomp \
+			--disable-libatomic \
+			--disable-libquadmath \
+			--disable-lto \
+			--disable-decimal-float \
+			--disable-libcc1 \
+			--disable-plugin \
+			--with-mpfr-include="${tmp_dir}/include" \
+			--with-mpfr-lib="${tmp_dir}/lib" \
+			--with-gmp-include="${tmp_dir}/include" \
+			--with-gmp-lib="${tmp_dir}/lib" \
+			--with-mpc-include="${tmp_dir}/include" \
+			--with-mpc-lib="${tmp_dir}/lib" \
+			--without-ppl \
+			--without-cloog \
+			--with-host-libstdcxx="-L${libdir} -L${tmp_dir}/lib -lstdc++ -lsupc++ -lm -lgmp -lgmpxx -lmpfr -lmpc" \
+			--with-build-sysroot="${dest_cross}" \
+			--with-sysroot="${dest_cross}" \
+			--with-native-system-header-dir="/usr/include"
 	fi
 }
 
 # Make build command
 atcfg_make() {
-	if [[ "${cross_build}" == "no" ]]; then
-		PATH=${at_dest}/bin:${PATH} \
-			${SUB_MAKE} all
-	else
-		echo "No make all on gcc_3 cross stage"
-	fi
+	PATH=${at_dest}/bin:${PATH} \
+		${SUB_MAKE} all
```

Comment:
Is this an unrelated change?

Reply:
No.  We didn't use to build gcc_3 on cross compiler. As we remove gcc_2, this is now mandatory.